packages/pieces/community/mailjet/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { createPiece, PieceAuth } from '@activepieces/pieces-framework';
+import { PieceCategory } from '@activepieces/shared';
+import { sendEmail } from './lib/actions/send-email';
+
+const mailjetAuth = PieceAuth.BasicAuth({
+  description: 'Enter your api credentials',
+  required: true,
+  username: {
+    displayName: 'API Key',
+    description: 'Enter your API Key here'
+  },
+  password: {
+    displayName: 'API Secret',
+    description: 'Enter your API Secret here'
+  }
+});
+
+export const mailjet = createPiece({
+  displayName: 'Mailjet',
+  description: 'Email delivery service for sending transactional and marketing emails',
+  auth: mailjetAuth,
+  minimumSupportedRelease: '0.20.0',
+  logoUrl: 'https://assets.mailjet.com/lib/images/mailjetLogo/mj_logo_only_icon_color.svg',
```

Comment:
Uploaded to `https://cdn.activepieces.com/pieces/mailjet.svg` can you please update the url